src/w3c/defaults.js

```diff
@@ -52,6 +53,7 @@ const w3cDefaults = {
     "no-headingless-sections": true,
     "privsec-section": true,
     "no-http-props": true,
+    "check-period-in-p": true,
```

Comment:
we would initially want false here... till we were sure it works well across a bunch of specs.